include/nl-priv-static-route/nl-priv-static-route.h

```diff
@@ -3,6 +3,10 @@
 #ifndef NETLINK_ROUTE_UTILS_PRIV_H_
 #define NETLINK_ROUTE_UTILS_PRIV_H_
 
+#include <netlink/route/link/bridge.h>
+
 extern const uint8_t *const _nltst_map_stat_id_from_IPSTATS_MIB_v2;
+extern int bridge_fill_vlan_info(struct nl_msg *msg,
```

Comment:
please give code in our headers a "nl" prefix. In this case, `_nl_bridge_fill_vlan_info()`.